src/OSPSuite.Core/Domain/UnitSystem/MolWeightDimensionConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using OSPSuite.Assets;
+
+namespace OSPSuite.Core.Domain.UnitSystem
+{
+   public abstract class MolWeightDimensionConverter : IDimensionConverter
```

Comment:
Also added MolWeight conversion capabiloty (general code here in core, specific code in the ospuite.r.dll)